buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -343,7 +343,7 @@ class ClusterFormationTasks {
         if (minimumMasterNodes > 0) {
             esConfig['discovery.zen.minimum_master_nodes'] = minimumMasterNodes
         }
-        if (node.config.numNodes > 1) {
+        if (node.config.minimumMasterNodes > 1) {
```

Comment:
Isn't `node.config.minimumMasterNodes` a closure? We just called it a few lines up, why not re-use the result?

Reply:
@DaveCTurner yikes sorry about that ... multitasking in action :) => reused the var.